app.py

```
@@ -30,9 +30,12 @@
 TARGET_LANG = os.environ.get("NEWS_TARGET_LANG", "ru")
 CACHE_TTL = int(os.environ.get("NEWS_CACHE_TTL", 15 * 60))  # seconds
 ITEMS_PER_SOURCE = int(os.environ.get("NEWS_ITEMS_PER_SOURCE", 50))
+VIEW_MODEL_WORKERS = int(os.environ.get("NEWS_VIEW_MODEL_WORKERS", 10))
 CACHE_DIR = pathlib.Path(os.environ.get("NEWS_CACHE_DIR", "/tmp/rss_cache"))
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/rss_cache"

(S108)